utils/ssl_renewel_util/renew_ssl_certs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Script to renew ssl certificates..
+
+# Prompt the user for the existing ssl certs expiry date for backing up the Dir with specific date.
+read -p "Enter the expiry date of existing ssl certs (e.g., 2024-12-31): " expiry_date
+
+# Define the source and destination directories
+src_dir="/etc/letsencrypt/live"
+dest_dir="/etc/letsencrypt/live-${expiry_date}"
+
+# Check if the source directory exists
+if [ -d "$src_dir" ]; then
+    # Move the directory
+    sudo mv "$src_dir" "$dest_dir"
+    echo "Directory moved to $dest_dir"
+else
+    echo "Source directory $src_dir does not exist."
+fi
+
+# Regenerate the ssl certificates
+# Prompt the user for the domain name
+read -p "Enter the domain name (e.g., sandbox.xyz.net): " mosip_domain
```

Comment:
check for the existence of `mosip_domain`, if not found prompt it

Reply:
for ssl-renew logger:

```
sudo mv /tmp/ssl-renew.log /tmp/ssl-renew.log.old || true
LOG_FILE="/tmp/ssl-renew.log"
ENV_FILE_PATH="/etc/environment"
source $ENV_FILE_PATH
```

Reply:
updated